src/goMain.ts

```diff
@@ -72,6 +72,11 @@ export function activate(ctx: vscode.ExtensionContext): void {
 		testCurrentFile(goConfig['testTimeout']);
 	}));
 
+	ctx.subscriptions.push(vscode.commands.registerCommand('go.test.previous', () => {
+		let goConfig = vscode.workspace.getConfiguration('go');
+		testPrevious();
```

Comment:
Do you intend to pass `testTimeout` here as in other test commands?  Or to not use `goConfig`?